smoke-test/pom.xml

```
@@ -634,6 +639,18 @@
       <version>${project.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.opennms.features.device-config.persistence</groupId>
+      <artifactId>org.opennms.features.device-config.persistence.impl</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+      <dependency>
+          <groupId>org.opennms.features.device-config</groupId>
+          <artifactId>org.opennms.features.device-config.rest</artifactId>
+          <version>30.0.0-SNAPSHOT</version>
```

should be project.version
```suggestion
          <version>${ project.version}</version>
```

---

Done